serverless.yml

```diff
@@ -32,6 +32,12 @@ functions:
   loadRdb:
     handler: gov.usgs.wma.waterdata.groundwater.BuildRdbFile
     role: arn:aws:iam::${self:custom.accountNumber}:role/csr-Lambda-Role
+    events:
+      - schedule:
+          name: etl-discrete-groundwater-rdb-${self:provider.stage}-event
+          description: "Schedule event to generate RDB file by invoking etl-discrete-groundwater-rdb lambda"
+          rate: cron(0 0 1 * ? *)
```

Comment:
@duselmann this cron will have to change, but for now it's a good placeholder to test a cron schedule of some kind is generated by serverless.

Reply:
How are we passing in the location folder? Are we just triggering it on a schedule and it should loop through them all? We have not discussed this yet. The current script expects a location folder but it is easily updated to process them all.